daemon/main.go

```diff
@@ -698,26 +698,33 @@ func initEnv(cmd *cobra.Command) {
 	log.Infof("Container runtimes being used: %s", workloads.GetRuntimesString())
 }
 func runDaemon() {
+	log.Info("Initializing daemon")
 	d, err := NewDaemon(config)
 	if err != nil {
 		log.WithError(err).Fatal("Error while creating daemon")
 		return
 	}
 
-	policy.Init()
+	policy.InitReserved()
+
+	log.Info("Starting connection tracking garbage collector")
 	endpointmanager.EnableConntrackGC(!d.conf.IPv4Disabled, true)
 
 	if enableLogstash {
+		log.Info("Enabling Logstash")
 		go EnableLogstash(logstashAddr, int(logstashProbeTimer))
 	}
 
+	log.Info("Launching node monitor daemon")
 	go d.nodeMonitor.Run(path.Join(defaults.RuntimePath, defaults.EventsPipe))
 
 	// Launch cilium-health in the same namespace as cilium.
+	log.Info("Launching cilium health daemon")
```

Comment:
cilium -> Cilium

Reply:
I assumed you also preferred the other lowercase ciliums switched, so I did. I hope that works!

Reply:
Yes. Thank you! :)